core/iwasm/common/gc/gc_object.c

```diff
@@ -698,6 +709,36 @@ wasm_object_get_ref_list(WASMObjectRef obj, bool *p_is_compact_mode,
         *p_ref_list = NULL;
         return true;
     }
+#if WASM_ENABLE_STRINGREF != 0
+    else if (rtt_type->type_flag == WASM_TYPE_STRINGREF) {
```

Comment:
Had better change to below to simplify the code:
```C
    else if (rtt_type->type_flag == WASM_TYPE_STRINGREF
             || rtt_type->type_flag == WASM_TYPE_STRINGVIEWWTF8
             || rtt_type->type_flag == WASM_TYPE_STRINGVIEWWTF16
             || rtt_type->type_flag == WASM_TYPE_STRINGVIEWITER) {
```

Reply:
Got it~